tutorials/wallet/sparrow/es.md

```diff
@@ -1,7 +1,7 @@
 ---
-name: Sparrow
+nombre: Sparrow
 
-description: Guía completa sobre Sparrow
+descripción: Guía completa sobre Sparrow
```

Comment:
These keys shouldn't be translated (metadata object of the tutorial)

Reply:
directly fixed it on the file.